.github/workflows/image.yaml

```diff
@@ -251,7 +251,7 @@ jobs:
           export PATH=$PATH:$GOPATH/bin
           export CREATE_VM=true
           export FLAVOR=${{ matrix.flavor }}
-          ./.github/run_test.sh "install-test"
+          go run github.com/onsi/ginkgo/v2/ginkgo --label-filter "install-test" --fail-fast -r ./tests/
```

Comment:
this uses the ginkgo version provided in the go.mod file, so no more downgrades or upgrades and adding deps in the targets.